rust/rsc/src/rsc/read_job.rs

```diff
@@ -79,8 +127,12 @@ pub async fn read_job(
                         output_symlinks,
                         output_dirs,
                         output_files,
-                        stdout_blob_id: matching_job.stdout_blob_id,
-                        stderr_blob_id: matching_job.stderr_blob_id,
+                        stdout_blob: resolve_blob(matching_job.stdout_blob_id, txn, &blob_stores)
+                            .await
+                            .unwrap(),
+                        stderr_blob: resolve_blob(matching_job.stderr_blob_id, txn, &blob_stores)
+                            .await
+                            .unwrap(),
```

Comment:
What happens if these blobs aren't resolvable? It seems like we will still return a match even though we don't have all the data needed.

Reply:
ah lol no, its actually much worse. `unwrap()` crashes the server on `Err` honestly we should probably just have a "no unwrap ever" policy in the RSC. I was just ignoring the error here, don't remember why, probably because its more code to handle it. I'll fix it